cockroachcloud/configure-cloud-org-sso.md

```diff
@@ -0,0 +1,215 @@
+
+## Plan to enable Cloud Organization SSO
+
+To ensure a smooth migration to Cloud Organization SSO, review the following information before you enable the feature.
+
+{{site.data.alerts.callout_info}}
+After it is enabled, Cloud Organization SSO cannot be disabled. To configure Cloud Organization SSO to behave like [Basic SSO](cloud-org-sso.html#basic-sso), refer to [Emulate Basic SSO](#emulate-basic-sso)
+{{site.data.alerts.end}}
+
+### Decide your custom URL
+
+When you [enable Cloud Organization SSO](#enable-cloud-organization-sso), you will configure a custom URL, that your users use to access {{ site.data.products.db }}. After the feature is enabled, your users must use that page to sign in. Your custom URL must be unique within {{ site.data.products.db }}. Communicate the custom URL to your users ahead of time.
+
+To change your custom URL after Cloud Organization SSO has been enabled, contact your account team.
+
+### Decide which authentication methods to enable
+
+All enabled authentication methods appear on your custom URL and are available to your users. By default, when you enable Cloud Organization SSO, the following authentication methods are enabled by default:
+
+- Email and password
+- Github
+- Google
+- Microsoft
+
+In addition, you can create authentication methods that connect to your identity provider using the [Security Access Markup Language (SAML)](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language) and [OpenID Connect (OIDC)](https://openid.net/connect/) identity protocols.
+
+Members are identified by their email address. To allow your members to select from multiple SSO authentication methods, ensure that the email addresses match across all of them. To allow members to migrate from password authentication to SSO, ensure that their email addresses match those in your identity providers.
+
+### Prepare for migration of existing members
+
+Keep the following in mind and communicate it to your members:
+
+- If your organization includes members who are also members of other organizations, and this is the first of their organizations to enable Cloud Organization SSO, those members must be re-added to your organization. If they sign in using an authentication method with [autoprovisioning](#autoprovisioning) enabled, they are automatically added upon successful sign-in. **However, those who were previously admins must be granted the role again**.
+- After you enable Cloud Organization SSO, all other members of your {{ site.data.products.db }} organization who were using [Basic SSO](cloud-org-sso.html#basic-sso) rather than an email and password must sign in again to regain access to your organization. After signing in, members retain the same access they had before the migration.
+
+During enablement of the feature, a list of affected members is shown, and those members are also notified individually.
+
+### Communicate to your members
+
+Communicate to your users about:
```

Comment:
It seems easiest to fold the 2 bullets in the preceding header into this section. This is assuming everything in the preceding header describes a point that needs to be communicated to the user. In any case, the two headings somewhat overlap as written.

Reply:
Good idea. Done.